app/searchfunctions.py

```diff
@@ -30,11 +29,12 @@ def get_all_assignment_status():
                         auth=HTTPBasicAuth(FSDR_USER, FSDR_PASS))
 
 
-def get_employee_records(user_filter=""):
+def get_employee_records(user_filter="", calledFromIAT=False):
     employee_record_url = URL(
-        FSDR_URL + f'/fieldforce/byType/byRangeAndUserFilter/').with_query(
+        FSDR_URL + f'/fieldforce/byType/byRangeAndUserFilter' + str("Iat/" if calledFromIAT == True else "/") ).with_query(
```

Comment:
`calledFromIAT == True` isn't required, you can just use `calledFromIAT`, since it's a boolean.

Reply:
Done